src/main/java/vendingmachine/model/InsertedMoney.java

```diff
@@ -0,0 +1,28 @@
+package vendingmachine.model;
+
+public class InsertedMoney {
+    private int insertedMoney;
+
+    public InsertedMoney(String money) {
+        isValidMoney(money);
+        this.insertedMoney = Integer.parseInt(money);
+    }
+
+    private void isValidMoney(String money) {
+        if (!money.chars().allMatch(Character::isDigit)) {
```

Comment:
```suggestion
        if (!money.chars()
        .allMatch(Character::isDigit)) {
```
가독성을 위해 라인을 분리해줘요! 그리고 기능을 분리해보는건 어떨까요?

Reply:
넵!
그런데 보통 문자열 자체의 내장 메서드를 통해서 단일 기능을 수행하는 경우도 함수로 분리하는 것이 나을까요?

메서드로 분리하면 함수 이름을 통해서 어떤 기능을 하는지 더 명확하게 표현하는 이점이 있을 것 같습니다!

Reply:
한 가지의 기능이라고 생각한다면 분리하는게 좋다고 생각합니다.
현재는 문자열 방식으로 검증하지만 이 방식이 다른 형태로 변경될 수 있다는 상황을 고려해볼 수 있겠네요. 😁

Reply:
아, 다른 방식으로 검증할 경우 형태가 변경된다는 점을 인지하지 못하고 있었네요!

문자열 자체 내장 메서드로 검증을 하다가, 검증 방법이 바뀔 경우 isValidMoney 메서드 자체를 손봐야 하지만
검증 기능을 메서드로 분리할 경우
1. 검증 기능 메서드만을 수정하거나
2. 다른 방식의 기능을 수행하는 메서드로만 바꿔주면 된다는 장점이 있을 것 같습니다.